ts/Core/Tooltip.ts

```diff
@@ -1370,8 +1376,14 @@ class Tooltip {
         } else {
             // update text
             if (tooltip.split) {
-                this.renderSplit(text as any, splat(pointOrPoints));
-            } else {
+                this.renderSplit(text as any, points);
+            } else if (
+                chart.polar ||
+                (
```

Comment:
Shouldn't we check, if point's series has axes? This might help with current problems of tooltip not displaying for some series not using axes

Reply:
Its just going to check if its within the plot if there are no axes, but which current problems?

Reply:
It looks that the tooltip is not displaying for Windbarb. Also, it is not displaying for Sankey in some specific cases (but maybe it was planned :) )

Reply:
It looks that the tooltip is not displaying for Windbarb. Also, it is not displaying for Sankey in some specific cases (but maybe it was planned :) )

Reply:
I think the visual diffs are mostly fine now, they are either related to points that are outside or due to the inverted and node fixes in `getChartCoordinatesFromPoint`.

Reply:
Ok, that's good. I would only check the windbarb graph - just in case (points in windbarb are rendered outside of axis extremes if I remember correctly). Apart of that all is good :)